packages/flutter/test/material/text_field_test.dart

```diff
@@ -2709,6 +2765,7 @@ void main() {
     //  ---------   rowBottomY
 
     final double rowBottomY = tester.getBottomLeft(find.byType(Row)).dy;
+    // Strut is not vertically rounded, so use actual values here instead of rowBottomY
```

Comment:
Confusing since we seem to be using rowBottomY here

Reply:
Oops, left over from a previous version!